src/components/Article.jsx

```diff
@@ -1 +1,17 @@
-// export function Article() {}
+import React from 'react';
+
+export function Article({ title, text, date }) {
+  return (
```

Comment:
You don't need return (but it's optional)

Reply:
`You don't need return (but it's optional)`
Why ?